symengine/tests/test_var.py

```diff
@@ -38,6 +38,13 @@ def test_var():
     # check return value
     assert v == [d, e, fg]
 
+# This works with nosetests,
+# but not with py.test, unclear why.
+# nosetests ignores pytest.mark.xfail so
+# we can safely decorate this function with this
+import pytest
+@pytest.mark.xfail
+def test_var_global_namespace():
     # see if var() really injects into global namespace
     raises(NameError, lambda: z1)
     _make_z1()
```

Comment:
Yes, this is the very reason for why I implemented the workarounds to be able to use nosetests (contrary to what we agreed upon). Can we save the transition to `py.test` for a later PR?